src/core/graph.test.js

```diff
@@ -943,80 +943,30 @@ describe("graph", () => {
         return [].concat(edgeGraphs, nodeGraphs);
       }
 
-      it("conservatively recomposes a neighborhood decomposition", () => {
+      it("recomposes a neighborhood decomposition", () => {
         const result = neighborhoodDecomposition(
           demoData.advancedMealGraph()
-        ).reduce((g1, g2) => Graph.mergeConservative(g1, g2), new Graph());
+        ).reduce((g1, g2) => Graph.mergeConservative([g1, g2]), new Graph());
```

Comment:
I believe using reduce here is inefficient / a bad pattern, and only exists because we didn't have the current mergeConservative api when written. should be `Graph.mergeConservative(neighborhoodDecomposition(`

Reply:
Absolutely. Fixed.